src/cart/crt_iv.c

```diff
@@ -71,6 +71,12 @@ struct crt_ivns_id {
 
 /* Structure for storing/passing of global namespace */
 struct crt_global_ns {
+
+<<<<<<< HEAD
+/* ALEX: test2 */
+=======
```

Comment:
(style)  spaces required around that '==' (ctx:ExO)

(style)  spaces required around that '==' (ctx:OxO)

(style)  spaces required around that '==' (ctx:OxO)

(style)  spaces required around that '=' (ctx:OxE)